migration/sql-test-files/081-query-empty-title.sql

```diff
@@ -0,0 +1,9 @@
+DELETE FROM spaces where id='171d52ff-fa00-46d7-ac24-94269908ad7a';
+INSERT INTO spaces (id, name) VALUES ('171d52ff-fa00-46d7-ac24-94269908ad7a', 'test space');
+
+-- empty query name
+-- this should fail
+DELETE FROM queries;
+INSERT INTO queries(fields, space_id, creator) 
```

Comment:
You're not testing that the name is empty unless you pass it as `''`. In this test you check that is cannot be `NULL`.